addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,647 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="bridge">
+		<label>RFXCOM Transceiver</label>
+		<description>This is universal RFXCOM transceiver bridge for manual configuration purposes.</description>
+
+		<config-description>
+			<parameter name="serialPort" type="text">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>Serial port where RFXCOM transceiver is connected.</description>
+				<required>true</required>
```

Comment:
please move the required to an attribute of the parameter (throughout this file)